tabbycat/participants/emoji.py

```diff
@@ -1570,6 +1570,126 @@ def populate_code_names_from_emoji(teams, overwrite=True):
     ("🧼",	True , "Soap"),
     ("🧽",	True , "Fun sponge"),
     ("🧯",	True , "Anti-fire Can"),
+    
+    # Unicode Version 12.0
+    ("🥱",	True , "Yawning Face"),
+    ("🤎",	False, "Brown Heart"),                      # too similar to another
+    ("🤍",	False, "White Heart"),                      # too similar to another
+    ("🤏",	True , "Pinching Hand"),
+    ("🦾",	False, "Mechanical Arm"),                   # potentially offensive
+    ("🦿",	False, "Mechanical Leg"),                   # potentially offensive
+    ("🦻",	False, "Ear with Hearing Aid"),             # potentially offensive
+    ("🧏",	False, "Deaf Person"),                      # potentially offensive
+    ("🧍",	False, "Person Standing"),                  # too similar to another
+    ("🧎",	False, "Person Kneeling"),                  # dull
+    ("🦧",	False, "Orangutan"),                        # potentially offensive
+    ("🦮",	True , "Guide Dog"),
+    ("🦥",	True , "Lazy Tree Dog"),
+    ("🦦",	True , "Water Dog"),
+    ("🦨",	True , "Stinky dog"),
+    ("🦩",	True , "Pink Dog"),
+    ("🧄",	False, "Garlic"),                           # dull
+    ("🧅",	False, "Onion"),                            # dull
+    ("🧇",	True , "Waffle"),
+    ("🧆",	True , "Falafel"),
+    ("🧈",	True , "Butter"),
+    ("🦪",	True , "Oyster"),
+    ("🧃",	True , "Beverage Box"),
+    ("🧉",	False, "Mate"),                             # too similar to another
+    ("🧊",	True , "Really Cold Cuboidal Figure"),
```

Comment:
Feel like the name is a bit long... how about "Cold Cuboid" or "Melty Cube" or something?